.github/workflows/ecli-image.yml

```diff
@@ -3,8 +3,6 @@ name: Build and public ecli docker image
 on:
   push:
     branches: "master"
-    paths:
-      - "VERSION"
```

Comment:
What is the reason for removing this?

Reply:
I think maybe we can publish the docker image automaticly once the CI has passed? Keep it the same as the auto release binaries...

Reply:
It seems reasonable, so that when the code repository changes, we can find out if there is a problem with the build process of the docker image in time, rather than when the VERSION file is modified. 

But on the other hand, the docker build process is a bit time consuming.